presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+To increase docker memory go to docker preferences > advanced > memory, and increase it appropriately.
```

Comment:
Suggestion:
"To increase docker memory, go to Docker Preferences -> Advanced -> Memory."